### ISSUE TITLE
Create a web hook in github to alert dev and production servers of update

### ISSUE THREAD

**Issue by dneiss** [thevwc/updateWebApps#4]
We could have the servers look to see if a new version of code is available and then automatically deploy it, further simplifying the deployment process. We could use a new release tag to indicate whether to update the app on the dev or production servers. For example, we could tag with an X.Y.Z tag and if a tag is found to be DEV-X.Y.Z then deploy to dev and PROD-X.Y.Z could deploy to production.

**Comment by HZass** [comment]
David
This could be very dangerous.  I think it is best to use an automated
process that is initiated manually.

Regards
Hans

**Comment by dneiss** [comment]
Agreed, we would need to think it through to ensure its safe. I believe that github can be configured to user certain security protocols to ensure that the source is trusted (OAuth) and such. BTW, the updateApps archives the currently running app before doing the update, so it can always be rolled back.
